### DISCUSSION TITLE
How to override a default alias in Windows with a custom profile method?

### DISCUSSION THREAD

**Discussion by yash-garg** [PowerShell/PowerShell#19540] [Q&A] [answered]
For example: There is a default alias for `gcm` which corresponds to `Get-Command`. Is there any way to change this alias forcefully to a separate method? (naming the function same doesn't work)

**Comment by rhubarb-geek-nz** [comment]
Are you able to use Set-Alias or Remove-Alias then New-Alias ?

**Reply by yash-garg** [reply]
Basically I have a function named `gcm` which has a default alias in powershell, but it doesn't call the function I wrote.

I'll try using `Remove-Alias` first then setting the function with same name.

**Comment by kasini3000** [comment]
Export-Alias -Path C:\scripts\aliases.txt
Import-Alias -Path C:\scripts\aliases.txt

**Comment by doctordns** [comment, marked as answer]
First naming a function using a built in alias is not best practice. And not naming it using verb-noun is also sub optimal.

You can put your function into a module and call the function using a <modulename>\<function name> syntax.

**Reply by yash-garg** [reply]
Okay, so overriding the default aliases is not a good choice.

**Reply by doctordns** [reply]
In generally, no!
